src/main/dialogs/quickSetStrip/qssTooltipDialog.js

```diff
@@ -173,12 +222,15 @@ gpii.app.qssTooltipDialog.getTooltipPosition = function (that, btnCenterOffset)
  * button.
  */
 gpii.app.qssTooltipDialog.show = function (that, btnCenterOffset) {
-    var offset = gpii.app.qssTooltipDialog.getTooltipPosition(that, btnCenterOffset);
+    var offsetAndDirection = gpii.app.qssTooltipDialog.getTooltipPosition(that, btnCenterOffset);
 
     that.dialog.setAlwaysOnTop(true);
 
     // reposition window properly
-    that.setPosition(offset.offsetX, offset.offsetY);
+    that.setPosition(offsetAndDirection.offsetX, offsetAndDirection.offsetY);
+
+    // apply the new arrow direction
+    that.applier.change("arrowDirection", offsetAndDirection.direction);
```

Comment:
Given you have folded all of this into the "show" action there doesn't seem to be a need to modelise this any more